README.md

```diff
@@ -156,7 +156,27 @@ class CreateUser
 end
 ```
 
-You'll get an `ArgumentError` if you pass any params not defined in the `inputs_for` block, or if you don't supply all required params. This is completely opt-in so if you don't provide an `inputs_for` block, no checking is performed.
+### Specifying types and nullable fields
+
+You can now also optionally specify types and nullability for your inputs. For the `type:` option, any ruby class can be used.
+
+```ruby
+inputs_for(:call) do
+  required :name, type: String, null: false
```

Comment:
I can appreciate where you're coming from. The only perspective is dispute is that I think `null: false` should be the default, because of what "required" means. Thus if you want to allow `nil` to be passed, it needs to be declared explicitly. While allowing the passing of `nil` is the rule, you are defining a new subset of rules here. Params  can pass nil, but `params.require(:foo)` doesn't need `null: false` to complicate it.

I know you want to be expressive/explicit, but this way feels redundant.